templates/download/server/download.html

```diff
@@ -0,0 +1,133 @@
+{% extends "download/_base_download.html" %}
+
+{% block title %}Thanks for downloading Ubuntu Server{% endblock %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1S5zY7HN_Q8eQWch0mCEfLICfj8pW1bct9_wGArCNIqc/edit{% endblock meta_copydoc %}
+
+{% block head_extra%}
```

Comment:
Thought this might affect the block evaluating - it doesn't! - but good to do anyway.
```suggestion
{% block head_extra %}
```